### PR TITLE
chore: add script to deprecate solar theme when release

### PR DESCRIPTION
**Pull request by bualoy-napat**
## Description

When a new version of Solar Theme has been published, the version should be deprecated.

Fixes # (issue)
DME-23123

## Checklist

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: .github/workflows/pre_release.yml [hunk 1]
```diff
@@ -49,3 +49,8 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Deprecate Solar Theme on npm
+        run: npm deprecate @refinitiv-ui/solar-theme ""
```
Comment (dtanp-rft) [inline .github/workflows/pre_release.yml]: Can we add description instead of leave it blank? ""

**Review by dtanp-rft — DISMISSED** [review]

File: .github/workflows/pre_release.yml [hunk 2]
```diff
@@ -49,3 +49,8 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Deprecate Solar Theme on npm
+        run: npm deprecate @refinitiv-ui/solar-theme "Solar theme no longer supported. Use @refinitiv-ui/halo-theme."
```
Comment (wattachai-lseg) [inline .github/workflows/pre_release.yml]: The message should be "Solar theme is deprecated. Consider migrating to @refinitiv-ui/halo-theme instead."

File: .github/workflows/prod_release.yml [hunk 3]
```diff
@@ -54,5 +54,10 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+      - name: Deprecate Solar Theme on npm
+        run: npm deprecate @refinitiv-ui/solar-theme "Solar theme no longer supported. Use @refinitiv-ui/halo-theme."
```
Comment (wattachai-lseg) [inline .github/workflows/prod_release.yml]: The message should be "Solar theme is deprecated. Consider migrating to @refinitiv-ui/halo-theme instead."

**Review by wattachai-lseg — CHANGES_REQUESTED** [review]

File: .github/workflows/pre_release.yml (hunk 1 above)
Comment (wattachai-lseg) [reply .github/workflows/pre_release.yml]: marking outdated comment as resolved.

**Review by wattachai-lseg — APPROVED** [review]

**Pull request merged**